app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, counter_cache: true
+end
```

Comment:
The method that incrememnts count is present in rails . it will automatically increase the count of like on post.

belongs_to :post, **counter_cache: true**